Online/Core/Teams/PreventDeadStranding.asm

```diff
@@ -0,0 +1,47 @@
+################################################################################
+# Address: 0x8016be28 # Pause_CheckButtonInputsToPause right after checking if
+# player is in sleep after death
+################################################################################
+
+.include "Common/Common.s"
+.include "Online/Online.s"
+
+.set REG_IS_ASLEEP, 14
+.set REG_ODB_ADDRESS, 15
```

Comment:
These registers don't get backed up by backup/restore. Use registers 20-31?

Reply:
Though frankly personally for this function I wouldn't even use backup/restore. Just avoid using r3 (you aren't doing any function calls so that's fine), and use r4-r12 for everything, including loading ODB_ADDRESS.

If you don't use r3 you don't need to worry about backing it up and restoring it

Reply:
Ooooops! 

Sounds good, will do.